chainer/backends/intel64.py

```diff
@@ -38,9 +38,15 @@ def check_ideep_available():
     Otherwise it raises ``RuntimeError``.
     """
     if _ideep_version is None:
+        # If the error is missing shared object, append a message to
+        # redirect to the ideep website.
+        msg = str(_error)
+        if 'cannot open shared object file' in msg:
+            msg += ('\n\nEnsure ideep requirements are satisfied: '
```

Comment:
Are `ideep` and `iDeep` used for different purposes?

Reply:
Fixed.